plugins/status.js

```diff
@@ -0,0 +1,241 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const async = require('async');
+
+exports.register = function () {
+    this.outbound = require('./../outbound');
+    this.queue_dir = require('./../outbound/queue').queue_dir;
+}
+
+exports.hook_capabilities = function (next, connection) {
+
+    if (connection.remote.is_local) {
+        connection.capabilities.push('STATUS');
+    }
+
+    next();
+}
+
+exports.hook_unrecognized_command = function (next, connection, params) {
+    if (params[0] !== 'STATUS') {
+        next();
+    }
+    else if (!connection.remote.is_local) {
+        return next(DENY, "STATUS not allowed remotely");
+    }
+    else {
+        return this.run(params[1], function (err, result) {
+
+            if (err) return next(DENY, err.message);
+            return connection.respond(211, result ? JSON.stringify(result) : "null", () => next(OK));
+        })
+    }
+}
+
+exports.run = function (cmd, cb) {
+    if (server.cluster && !cmd.match(/^QUEUE LIST/)) {
```

Comment:
```suggestion
    if (server.cluster && !/^QUEUE LIST/.test(cmd)) {
```

Match works here as you expect, but it creates an array and returns it with your match results. If you don't need the match results, `test` will return a boolean instead, which is more efficient.